src/modelmanager.cpp

```diff
@@ -634,6 +676,12 @@ Status ModelManager::reloadModelVersions(std::shared_ptr<ovms::Model>& model, st
 }
 
 Status ModelManager::reloadModelWithVersions(ModelConfig& config) {
+    auto model = getModelIfExistCreateElse(config.getName());
+    if (model->isAnyVersionSubscribed() && config.isDynamicParameterEnabled()) {
+        SPDLOG_LOGGER_DEBUG(modelmanager_logger, "Requested setting dynamic parameters for model {} but it is used in pipeline", config.getName());
```

Comment:
I think this deserves WARNING or at least INFO.

Reply:
Done